### DISCUSSION TITLE
Daily 10/04

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#21] [Daily Sprint]
Ayer tuvimos nuestra weekly en la que repasamos lo por hacer en la semana. 
Estuve leyendo los materiales de la plataforma del ISPC y completando algunas tareas. 
Además, estuve viendo la documentación para poder empezar a subirla a la wiki.
Hoy por la tarde subiré los apuntes de la reunión de ayer, empezaré a subir la documentación a la wiki y seguiré leyendo los materiales de la plataforma.

**Comment by rpgrca** [comment]
Buenas! Hoy voy a armar el script de base de datos usando las tablas que propuso el ISPC. Creo que no hay issue sobre eso todavía así que voy a armar una antes de subirlo, y armar un branch también.

**Reply by AhumadaCeleste** [reply]
Si te resulta de utilidad para hacer el script, hoy hago el modelo entidad de relación el cual tengo ya pre-armado en lápiz. Por ello consultaba sobre la definición del proyecto a fin de considerar si nuestro sitio precisa sumar de algo especifico.

**Comment by AhumadaCeleste** [comment]
Ayer participe de la weekly, revise el material, subí el pdf. con el resultado de ingles y avance con Javascript. También diseñe un boceto de del modelo de entidad de relación para poder hacerlo hoy y subirlo.

**Comment by claubott** [comment]
No pude asistir a la weekly del domingo, pero compartieron la grabación y el lunes participé en la discusión sobre el tema del proyecto.

**Comment by KAAcevedo** [comment]
¡Lunes 10/04 18hs me sumé como nueva integrante de este proyecto!
Estoy poniéndome al día con todo, me queda pendiente ver la grabación de la reunión semanal del domingo pasado
